service/src/main/java/uk/gov/hmcts/reform/fpl/utils/PeopleInCaseHelper.java

```diff
@@ -21,4 +23,12 @@ public static String getFirstRespondentLastName(List<Element<Respondent>> respon
             .map(RespondentParty::getLastName)
             .orElse("");
     }
+
+    //To be deleted when print and post is available to use (before March 3rd)
+    public static List<String> formatRepresentativesForPostNotification(List<Representative> representatives) {
+        return representatives.stream()
+            .map(representative -> representative.getFullName() + "\n" + representative.getAddress().getAddressAsString(
```

Comment:
could extract the logic to build the address string to separate private method

Reply:
I'm happy keeping it here given it is not very complex

Reply:
Could use `String.format(...)`, might make it look a bit nicer

Reply:
changed, although personally don't think it looks much nicer